.devcontainer/Dockerfile

```
@@ -0,0 +1,32 @@
+
+# Copy uv and uvx (latest version)
+COPY --from=ghcr.io/astral-sh/uv:latest /uv /uvx /bin/
+
+# Set non-interactive frontend for apt
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Enable Docker BuildKit for cache mounts
+ENV DOCKER_BUILDKIT=1
+
+# Install system dependencies with cache mount for faster rebuilds (latest versions)
+RUN --mount=type=cache,target=/var/cache/apt \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)